packages/world-modules/src/modules/delegation/getSignedMessageHash.sol

```diff
@@ -24,7 +24,7 @@ function getSignedMessageHash(
   address worldAddress
 ) view returns (bytes32) {
   bytes32 domainSeperator = keccak256(
-    abi.encode(keccak256("EIP712Domain(uint256 chainId,address verifyingContract)"), block.chainid, worldAddress)
+    abi.encode(keccak256("EIP712Domain(address verifyingContract,bytes32 salt)"), worldAddress, bytes32(block.chainid))
```

Comment:
can we add some context here to explain why we use the salt, not the chain ID param, for this value?

Reply:
agreed, it is quite nonstandard!